shims/spark34/src/main/scala/org/apache/spark/sql/execution/datasources/v2/velox/DwrfScanBuilder.scala

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
@zhouyuan We have removed DwrfScan. Why need in Spark3.4?

Reply:
yes, it should be removed. it's accidently left during my rebase work.